spec/views/plantings/edit.html.haml_spec.rb

```diff
@@ -40,7 +40,6 @@
 
     it 'includes helpful links for crops and gardens' do
```

Comment:
if its only including a helpful link for garden now, maybe this name should be changed. or the assert_select should find the new request link.